app/src/main/res/raw-mcc510/abtesting.json

```diff
@@ -20,5 +20,37 @@
         "bucket_range_end": 20
       }
     ]
+  },
+  {
+    "name": "ab_testing_experiment_name_top_sites",
+    "enabled": true,
+    "newUserOnly": true,
+    "buckets": [
+      {
+        "experiment_name": "id_topsite_rr_0429_control",
+        "bucket_range_start": 1,
+        "bucket_range_end": 4
```

Comment:
Just want to confirm do we want different bucket range setting in Indonesia?

Reply:
yes we want different bucket range for india and indonesia
https://github.com/mozilla-tw/FirefoxLite/issues/4871#issuecomment-611850019